index.html

```diff
@@ -71,6 +73,7 @@ <h2 class="container job-title">I am a Software Developer</h2>
     <section class="works-header">
         <div class="works-title">
             <h1>My Recent Works</h1>
+            <hr>
```

Comment:
- [ ] Please, ensure that The Title `"My Recent Works"`  is aligned to the left and after there's a line that takes the rest of the page's width.
**1.You design**
![image](https://user-images.githubusercontent.com/80612925/171974732-db4d76f0-658c-4e7c-872d-8b58e2ad61c8.png)
2. Figma **design**
![image](https://user-images.githubusercontent.com/80612925/171974767-de8366d3-71df-4c57-9e42-fb55c41b1fa9.png)